go/felix/intdataplane/endpoint_mgr.go

```diff
@@ -207,8 +207,24 @@ func (m *endpointManager) resolveWorkloadEndpoints() error {
 			var ipStrings []string
```

Comment:
I've done something like the former of these - do you think that more is still advisable?